tests/embedding/test_graphics_export.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (C) 2022 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Graphics export tests"""
+
+import os
+
+import pytest
+
+try:
+    from ansys.mechanical.core import global_variables
+except:
+    # No embedding - this import breaks test collection
+    global_variables = {}
+
+from ansys.mechanical.core.examples import delete_downloads, download_file
+
+
+@pytest.mark.embedding
+@pytest.mark.parametrize("image_formats", ["PNG", "JPG", "BMP"])
+def test_image_export(printer, embedded_app, image_formats):
+    """Tests to check image export."""
+    printer(f"{image_formats} export")
+    globals().update(global_variables(embedded_app, True))
+    mechdb_file = download_file("graphics_test.mechdb", "pymechanical", "test_files")
```

Comment:
@dipinknair if we continue this pattern of associating a mechdb with a test can we please add a fixture where we can put the path to the mechdb in the decorator?